rbac/management/principal/it_service.py

```diff
@@ -78,13 +79,12 @@ def __init__(self):
         self.port = settings.IT_SERVICE_PORT
         self.protocol = settings.IT_SERVICE_PROTOCOL_SCHEME
         self.it_request_timeout = settings.IT_SERVICE_TIMEOUT_SECONDS
+        self.it_url = f"{self.protocol}://{self.host}:{self.port}{self.base_path}{IT_PATH_GET_SERVICE_ACCOUNTS}"
 
     @it_request_all_service_accounts_time_tracking.time()
-    def request_service_accounts(self, bearer_token: str) -> list[dict]:
+    def request_service_accounts(self, bearer_token: str, client_ids: list[str] = None) -> list[dict]:
```

Comment:
to make the pr check happy you can use here the `Optional` from `typing`
`client_ids: Optional[list[str]] = None`

Reply:
Changed, thank you!